src/Compilers/Core/MSBuildTask/CopyRefAssembly.cs

```diff
@@ -40,7 +40,7 @@ public override bool Execute()
 
             if (File.Exists(DestinationPath))
             {
-                Guid source;
+                var source = Guid.Empty;
```

Comment:
Bonus points if someone can guess why I had to do this.

Reply:
Because compiler can see private struct fields in new TFM?

Reply:
Close ... it's because the definition of `Guid` is different between netstandard1.3 and net46 according to the reference assemblies. In one case it's a struct without fields, in the other it's a struct with private fields.

Reply:
(That's basically what I meant.)